src/components/Reviews.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+
+const Reviews = () => {
+	const [reviews, setReviews] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
+
+	useEffect(() => {
+		axios
```

Comment:
This should be extracted as a function into api.js, which is where all your api call functions will live moving forwards.